features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/MessagesView.kt

```diff
@@ -189,13 +195,8 @@ fun MessagesView(
                     roomAvatar = state.roomAvatar.dataOrNull(),
                     heroes = state.heroes,
                     roomCallState = state.roomCallState,
-                    onBackClick = {
-                        // Since the textfield is now based on an Android view, this is no longer done automatically.
-                        // We need to hide the keyboard when navigating out of this screen.
-                        localView.hideKeyboard()
-                        onBackClick()
-                    },
-                    onRoomDetailsClick = onRoomDetailsClick,
+                    onBackClick = { hidingKeyboard { onBackClick() } },
+                    onRoomDetailsClick = { hidingKeyboard { onRoomDetailsClick() } },
```

Comment:
I was thinking about also wrapping `onJoinCallClick` below but since it will start another Activity, I think it's fine to keep the keyboard opened.